corehq/util/sentry.py

```diff
@@ -62,6 +63,11 @@ def _get_rate_limit_key(exc_info):
                 return key
 
 
+def is_pg_cancelled_query_exception(e):
+    PG_QUERY_CANCELLATION_ERR_MSG = "canceling statement due to conflict with recovery"
```

Comment:
is there a reason to keep that constant in the method?
Can move that to `corehq/const.py` if that looks the right place.

Reply:
silly question, We assuming that that message can only come from standby since its a conflict with recovery or it can come from mater as well?

Reply:
It only comes from standby, masters don't cancel query with this message.